circuits-circom/regexes/hdfc/hdfc_payee_id.json

```diff
@@ -0,0 +1,16 @@
+{
+  "parts": [
+    {
+      "is_public": false,
+      "regex_def": "to VPA "
```

Comment:
Let's actually consolidate payee_id and acc_num into 1 regex string with 2 reveals

Reply:
I was thinking about it. Although why is the current one not good? This is a tradeoff between cleaner and abstracted code with more constraints vs not cleaner code with less constraints.